src/runtime/virtcontainers/kata_agent.go

```diff
@@ -368,7 +368,7 @@ func (k *kataAgent) init(ctx context.Context, sandbox *Sandbox, config KataAgent
 	imageRequestTimeout = time.Duration(sandbox.config.ImageRequestTimeout) * time.Second
 	k.Logger().WithFields(logrus.Fields{
 		"imageRequestTimeout": fmt.Sprintf("%+v", imageRequestTimeout),
-	}).Warn("The imageRequestTimeout has been set ", imageRequestTimeout, " seconds.")
+	}).Infof("The imageRequestTimeout has been set.")
```

Comment:
Can you merge this in the previous commit?

Reply:
Ok.